fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/NumericAggregationValue.java

```diff
@@ -208,6 +210,75 @@ private static Map<Pair<LogicalOperator, TypeCode>, PhysicalOperator> computeOpe
         return mapBuilder.build();
     }
 
+    /**
+     * BitMap aggregation {@code Value}.
+     */
+    public static class BitMap extends NumericAggregationValue implements StreamableAggregateValue, IndexableAggregateValue {
```

Comment:
Can you please rename this to the more standard-ish `BitmapConstructAgg[Value]`.